src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="/"
+    method="post"
+    class="searchBar searchBar-small"
+    data-qa="big"
+    >
+      <input
```

Comment:
checklist.rm from your root folder of homework

![image](https://user-images.githubusercontent.com/99652389/178436947-826372da-0681-4505-be1e-28679c02a366.png)